samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md

```diff
@@ -1,3 +1,7 @@
 # Role based access sample with Microsoft.Azure.EventHubs SDK #
 
 For more information on Role based access (RBAC) and how to run this sample follow [this](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-role-based-access-control) link.
```

Comment:
@spelluru 

The linked page does not talk about "how to run this sample". Maybe this was meant to be more like "how to assign roles"?

```suggestion
For information on Role based access (RBAC) in Event Hubs, and to assign roles, see [Authorize access to Event Hubs resources using Azure Active Directory](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-role-based-access-control).
```

Reply:
@spelluru gentle ping

Reply:
@lilyjma - sorry. what's the question?